js-miniapp-sample/src/services/uuid/types.js

```diff
@@ -0,0 +1,2 @@
+const SET_UUID = "SET_UUID";
```

Comment:
This is minor, but we can export a constant in one line:

```suggestion
export const SET_UUID = "SET_UUID";
```

Reply:
as the app grows, i assume there will be few more constants & flow types will be added.

Reply:
Yup definitely, and we are able to export each individual constants and types instead of bundling them all up.

Doing this:

```javascript
export const a = "a";
export const b = "b";
export const c = "c";
export const d = "d";

export type t = {...}
```

Instead of:

```javascript
const a = "a";
const b = "b";
const c = "c";
const d = "d";

type t = {...}

export {
  a,
  b,
  c,
  d
}

export type {
  t
}
```

Reply:
i feel consolidating all the exported constants looks clean, when we have some types (sub types) in a file which are not shared to any other file. what do you think?

Reply:
That sounds fine with me to consolidate all exports 👍